src/faq.md

```diff
@@ -219,6 +219,15 @@ Yes. Dart programs can be compiled to native x64 machine code for running in a
 Terminal/Command Prompt on desktop operating systems such as Windows, macOS, and
 Linux. For more details, see the [dart2native][] documentation.
 
+### Q. Which is faster; JIT- or AOT-compiled code?
```

Comment:
";" -> ":"

Or maybe:

### Q. Is AOT- or JIT-compiled code faster?

@kwalrath, WDYT?